pkg/imgpkg/bundle/bundle.go

```diff
@@ -48,11 +48,16 @@ func (o *Bundle) Repo() string      { return o.plainImg.Repo() }
 func (o *Bundle) Tag() string       { return o.plainImg.Tag() }
 
 func (o *Bundle) Pull(outputPath string, ui goui.UI, pullNestedBundles bool) error {
-	return o.pull(outputPath, ui, pullNestedBundles, "", map[string]bool{}, 0)
+	_, err := o.checkedImage()
+	if err != nil {
+		return err
+	}
```

Comment:
not sure why we need this section here, first few lines of o.pull does the same thing.

Reply:
ah i see because we use o.Repo()...